src/pages/layer-2.tsx

```diff
@@ -720,39 +681,10 @@ const Layer2Page = ({ data }: PageProps<Queries.Layer2PageQuery>) => {
         <p>
           <Translation id="layer-2-use-layer-2-generalized-1" />
         </p>
-        <CardGrid>
-          {layer2DataCombined
-            .filter((l2) => !l2.purpose.indexOf("universal"))
```

Comment:
We are missing this entire section which is dedicated to the "Generalized layer 2s".

From prod version:
![image](https://user-images.githubusercontent.com/468158/219204929-37ac02a9-0331-46be-a30f-0b17dd35fa2f.png)

Reply:
Whoops! Adding section comments was supposed to avoid this! 🤪

Reply:
I will also update the heading and paragraph tags like you requested for the Home page

Reply:
The columns are stretching to fill the space.

![image](https://user-images.githubusercontent.com/468158/220451785-a1660016-8446-4cf4-ada2-4e8a2193dbac.png)

we want to preserve the width of those columns. Check the prod version
![image](https://user-images.githubusercontent.com/468158/220451890-1ced68a7-bdcb-4e3f-aa3a-887bf897b7a4.png)


I guess we want to have the `280px` as the max width.

Reply:
Replaced `templateColumn` prop with `columns` with a value of `4` to take care that. So no need for an explicit max width! :)